homeassistant/components/matter/cover.py

```diff
@@ -136,12 +172,41 @@ def _update_from_device(self) -> None:
             self.current_cover_position,
         )
 
+        # current tilt position is inverted in matter (100 is closed, 0 is open)
```

Comment:
we can return early here if there is no tilt support ?

Reply:
This is already done in the second PR: 

- [condition for lift]( https://github.com/home-assistant/core/pull/92278/files#diff-4b2b22eb334827e09a3d8dfdd0e35111a6534495741a49727962c2186d6f3847R161) 
- [condition for tilt](https://github.com/home-assistant/core/pull/92278/files#diff-4b2b22eb334827e09a3d8dfdd0e35111a6534495741a49727962c2186d6f3847R181)